chapters/3-measurement.tex

```diff
@@ -37,125 +39,210 @@
 	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}}
-\index{master datasets}
-details all project-wide time-invariant information
-about all observations encountered,
-as well as their relationship to the research design,
-typically summarized by sampling and treatment status.
-Having a plan for how to get the raw data into analysis shape 
-before you acquire it,
-and making sure that the full research team knows where to find this information,
-will save you a ton of time during the course of the project
-and increase the quality of your research.
-
-You should create a master dataset
-for each unit of observation
-relevant to the research.
-This includes all units used for significant research activity,
-like data collection or data analysis.
-Therefore, any unit
-that will be used in sampling or treatment assignment,
-must have a master dataset,
-and that master dataset -- not field data --
-should be used when sampling or assigning treatment.
-Master data sets are often created from field data,
-but master data sets should be treated differently
-as there can be several field datasets,
-but only one authoritative master dataset.
-Master data sets should be created as soon as you
-start to get information about units.
-For example, when receiving a type of administrative data set for the first time,
-or after doing a respondent listing before a survey.
-
-You also need to record how all datasets for each unit of observation
-will link or merge with each other as needed.
-This linking scheme is called a \textbf{data map}\sidenote{
-	\url{https://dimewiki.worldbank.org/data\_map} (TO BE CREATED)}.
-\index{data maps} 
-A data map is more than just a list of datasets.
-Its purpose is to specify the characteristics and linkages of those datasets.
-To link properly, the master datasets must include fully unique ID variables.\sidenote{
-	\url{https://dimewiki.worldbank.org/ID\_Variable\_Properties}}
-The master datasets should indicate whether datasets should be merge one-to-one,
-for example, merging baseline data and endline data that use the same unit of observation,
+list all observations your project ever encounter
+are the authoritative source for all research data
+such as ID values, sampling and treatment status, etc.
+\textbf{Data flow charts}\sidenote{
+	\url{https://dimewiki.worldbank.org/Data\_Flow\_Chart}}
+list all datasets that are needed to create each analysis dataset,
+and how they should be created by merging, appending 
+or in any other way link different datasets.
+
+\subsection{Creating a data plan}
+
+The components of the data plan is the best tools a research team has for
+the lead researchers to communicate their vision for the data work, 
+and for the research assistants to communicate their understanding of that vision.
+While the time to create the data plan is before any data has been acquired,
+it should be a dynamic documentation, 
+that you should keep up to date as the project evolves.
```

Comment:
```suggestion
it should be a piece of documentation that is continuously updated as the project evolves.
```

Since we use "dynamic" for a specific technical use later